packages/local-environment/hardhat/hardhat.config.js

```diff
@@ -56,5 +64,9 @@ module.exports = {
       // url: "http://0.0.0.0:8545",
       accounts,
     },
+    forking: {
+      url: "https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}",
+      blockNumber: parseInt(process.env.BLOCK_NUMBER),
+    },
```

Comment:
Create a temp object, add forking into it on demand, pass it to the module.exports

Reply:
Hardhat has now been made into a temp object, and forking added on demand.